content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -192,6 +192,53 @@ The original destination type of the traffic is used to determine if a service o
 For instance, traffic which is addressed to a service, even though ultimately resolved to a pod IP, is always treated by the ambient mesh as to-service and would use a service-attached waypoint.
 {{< /tip >}}
 
+## Cross-namespace waypoint use {#usewaypointnamespace}
+
+Straight out of the box a waypoint proxy is usable by resources within the same namespace. Beginning with Istio 1.23 it is possible to use waypoints in different namespaces. In this section we will examine the gateway configuration required to enable cross-namespace use as well as how to configure your resources to use a waypoint from a different namespace.
+
+### Configure a waypoint for cross-namespace use
+
+In order to enable cross-namespace use of a waypoint, the `Gateway` should be configured to [allow routes](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1.AllowedRoutes) from other namespaces.
+
+{{< tip >}}
+The keyword, `All`, may be specified as the value for `allowedRoutes.namespaces.from` in order to allow routes from any namespace.
+{{< /tip >}}
+
+The following Gateway would allow resources in a namespace called "cross-namespace-waypoint-consumer" to use this egress-gateway:
+
+{{< text syntax=yaml >}}
+kind: Gateway
+metadata:
+  name: egress-gateway
+  namespace: common-infrastructure
+spec:
+  gatewayClassName: istio-waypoint
+  listeners:
+  - name: mesh
+    port: 15008
+    protocol: HBONE
+    allowedRoutes:
+      namespaces:
+        from: Selector
+        selector:
+          matchLabels:
+            kubernetes.io/metadata.name: cross-namespace-waypoint-consumer
+{{< /text >}}
+
+### Configure resources to use a cross-namespace waypoint proxy
+
+By default, the Istio control plane will look for a waypoint specified using the `istio.io/use-waypoint` label in the same namespace as the resource which the label is applied to. It is possible to use
+a waypoint in another namespace by adding a new label, `istio.io/use-waypoint-namespace`. `istio.io/use-waypoint-namespace` works for all resources which support the `istio.io/use-waypoint` label.
+Together, the two labels specify the name and namespace of your waypoint respectively. For example, to configure a ServiceEntry called "istio-site" to use a waypoint called "egress-gateway" in the namespace
+called "common-infrastructure" you could use the following commands:
```

Comment:
```suggestion
Together, the two labels specify the name and namespace of your waypoint respectively. For example, to configure a `ServiceEntry` named `istio-site` to use a waypoint named `egress-gateway` in the namespace
named `common-infrastructure`, you could use the following commands:
```

Reply:
same things here, `ServiceEntry` is a kind of resource so back ticks make sense. Seems like the others should just be double quotes though

Reply:
nm, we basically never make a distinction like that elsewhere